platform/tron/stake.go

```diff
@@ -12,6 +12,11 @@ import (
 
 const Annual = 0.74
 
+var unknownStakeValidator = blockatlas.StakeValidator{
+	ID:     "unknown",
```

Comment:
just realized - the client need ID as validators address to unstake. Can you specify address to ID?

Reply:
yes, ok